web/src/pages/UserEditPage.vue

```diff
@@ -29,7 +29,15 @@
       </div>
     </h2>
 
+    <AdminUserEditForm
+      v-if="isSystemAdmin"
+      class="mt-10"
+      :user-id="user.id"
+      :cancel-button-options="{ to: { name: 'UsersPage' } }"
+      @saved="refresh"
+    />
     <UserEditForm
+      v-else
```

Comment:
This page is already the user admin page, so I'd expect only the edit form here to be the "admin" one.

Only Admins should be able to access the "All Users" page, and only the all user's page grants you access to this UserEditPage.

See `web/src/components/base-layout/KebabMenu.vue`
```
<v-list-item
  v-if="isSystemAdmin"
  title="All Users"
  :to="{ name: 'UsersPage' }"
  prepend-icon="mdi-account-group"
/>
```

You could definitely argue that this should be called the AdminUserEditPage, and I'd would probably agree.
Normally, there might be a "read" view for all users that was publicly accessible, and the "Edit" button would only be shown to admins.